Code/Comscore/Source/integration/Enums.swift

```diff
@@ -14,6 +14,24 @@ public enum ComScoreUserConsent: String {
     case unknown = "-1"
 }
 
+@frozen
+public enum ComscoreUsagePropertiesAutoUpdateMode: String {
+    case foregroundOnly
```

Comment:
Maybe add a basic doc about these properties. (as this is a public API)

Reply:
✅